jstp/src/main/java/com/metarhia/jstp/connection/Connection.java

```diff
@@ -124,7 +124,7 @@
 
   /**
    * Creates connection instance over specified transport {@param transport}. Session restoration
```

Comment:
What do you think about shortening it to '{@link SessionRestorationPolicy} is used by default'?

Reply:
OK